cdap-app-templates/cdap-etl/cdap-etl-tools/pom.xml

```diff
@@ -71,7 +71,9 @@
     </resources>
     <plugins>
       <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-shade-plugin</artifactId>
+        <version>2.4.3</version>
```

Comment:
I copied the version from another pom.xml. It's the highest. Open to other suggestions.